lms/templates/verify_student/edx_ace/verificationsubmitted/email/body.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'ace_common/edx_ace/common/base_body.html' %}
+
+{% load i18n %}
+{% load static %}
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+  <tr>
+    <td>
+      <h1>
+        {% trans "ID Verification Submitted" as tmsg %}{{ tmsg | force_escape }}
+      </h1>
+      <p style="color: rgba(0,0,0,.75);">
+        {% filter force_escape %}
+          {% blocktrans %}Hello {{ full_name }},{% endblocktrans %}
+        {% endfilter %}
+      </p>
+      <p style="color: rgba(0,0,0,.75);">
+        {% filter force_escape %}
+           {% blocktrans %}Thank you for submitting your photos for identity verification with {{ platform_name }}. {% endblocktrans %}
+        {% endfilter %}
+      </p>
+
+      <p style="color: rgba(0,0,0,.75);">
+        {% trans "We have received your photos and they will be reviewed within 2-5 days." as tmsg %}{{ tmsg | force_escape }}
+
+      {% filter force_escape %}
+        {% blocktrans with "https://courses.edx.org/dashboard" as dashboard_link %}You may check the status on your dashboard: {{dashboard_link}} {% endblocktrans %}
```

Comment:
We shouldn't hard code the link. There are settings like LMS_ROOT_URL etc. that should be passed to template.

Reply:
Got it.